### DISCUSSION TITLE
Search syntax & API: Help needed

### DISCUSSION THREAD

**Discussion by macphil** [community/community#27885] [Apps, API and Webhooks] [answered]
I am trying to retrieve the runs created before a certain date via api.

According to [reference](https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-repository), the search syntax must be taken into account:

> created
Returns workflow runs created within the given date-time range. For more information on the syntax, see "[Understanding the search syntax](https://docs.github.com/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#query-for-dates)."

But I'm not able to apply them.
I tried the following without success (received always all runs):
- `/repos/{owner}/{repo}/actions/runs?created:<2022-07-01`
- `/repos/{owner}/{repo}/actions/runs?created<2022-07-01`

What have I missed?

**Comment by airtower-luna** [comment, marked as answer]
> What have I missed?

The `=` between parameter name and value in [URL parameters](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#parameters). :wink: 

This works for me: `/repos/{owner}/{repo}/actions/runs?created=<2022-05-15`

**Reply by macphil** [reply]
Thanks a lot!
a small char for the solution - a giant leap for me ;-)
